### ISSUE TITLE
Remake the Ethreal Bloom as a device

### ISSUE THREAD

**Issue by chrisbecke** [Azanor/thaumcraft-suggestions#22]
I might be alone in this, but the ethereal bloom does not really "fit" thematically.  They way they are used at least, seems more like a device - place it on a block, without any regards for the environment: it can be on netherrack or rock just as easily as dirt or farmland - it doesn't care where it grows - in a cave or a dimension without sunlight doesn't bother it.
It grows instantly, which is handy, as a thaumaturge with a taint problem on their hands generally doesn't have enough blooms to cover it.

Basically, in every way, it seems to act more like a placeable block/device, rather than a flower. It has no growth mechanic (surely a flower should interact with the hoe of growth in some way?).

**Comment by majijn** [comment]
so do normal minecraft flowers in general, I'm not sure I see the problem/disparity

**Comment by Hyxaru** [comment]
Atleast vanilla flowers require a minimum light level and a dirt type as a support block.

**Comment by Trentin996** [comment]
It's a magic freaking flower. It doesn't obey the traditional logic. That's magic for you.

**Comment by Argis13** [comment]
I just wish there was some botania compatibly with the bloom.
